README.md

```diff
@@ -38,13 +38,19 @@ Uploads a theme to the server. Requires the theme to have been previously synchr
 
 ### `discourse_theme rspec PATH`
 
-Runs the [RSpec](https://rspec.info/) system tests under the `spec` folder for the theme.
+Runs the [RSpec](https://rspec.info/) system tests under the `spec` folder in the designated directory.
```

Comment:
The first time I asked myself, which "desginated directory" and then realized it was the theme directory, not the spec or local installation directory. Do you think it is worth clarifying this?

Maybe it was just me.

```suggestion
Runs the [RSpec](https://rspec.info/) system tests under the `spec` folder in the designated theme directory.
```